### PR TITLE
Fix dapper build distribution

### PR DESCRIPTION
**Pull request by nickgerace**
Relevant issue: https://github.com/rancher/rancher/issues/30516

### CODE REVIEW

File: scripts/build [hunk 1]
```diff
@@ -53,3 +53,9 @@ fi
 if ! [ "$LOCAL" = "false" ]; then
     build-binaries ${OS} ${ARCH}
 fi
+
+# When only running "dapper build", you may have to copy the binaries to "dist".
+if [ "$COPY_TO_DIST" = "true" ]; then
```
Comment (StrongMonkey) [inline scripts/build:58]: Can this be the default behavior? When you run dapper build it is always going to cp to dist/artifacts

Comment (nickgerace) [reply scripts/build:58]: Sure thing, but we should remove it from `scripts/package` then. Thoughts?

Comment (StrongMonkey) [reply scripts/build:58]: yep, I think so

Comment (nickgerace) [reply scripts/build:58]: Ok great, will do

File: .drone.yml [hunk 2]
```diff
@@ -194,6 +196,8 @@ steps:
 # Update this tag once dapper arm v7 works on drone.
 - name: build
   image: rancher/dapper:v0.4.1
+  environment:
```
Comment (StrongMonkey) [inline .drone.yml:199]: Remove this?

**Review by StrongMonkey — APPROVED** [review]

**Review by luthermonson — APPROVED** [review]

**Pull request merged**